src/ESports.ts

```diff
@@ -124,9 +127,9 @@ export default class ESportsAPI {
 
     private sendPrintout(channel: Discord.TextChannel, data: Map<string, ESportsLeagueSchedule[]> | undefined, date: string) {
 
-        date = date.split(" ").join("/");
-        if (!data) {
-            channel.send(`No games played on ${date}.`);
+        date = (date || momentjs().format("YYYY M D")).split(" ").join("/");
+        if (!data || data.size === 0) {
```

Comment:
`data.length`, but `data` will already evaluate to false if the length is `0`, so this is duplicate and won't be executed.

Reply:
data is a Map, and an empty map does not evaluate to false. Thus, this will not be changed.